crates/hotshot/src/traits/election/static_committee.rs

```diff
@@ -124,6 +123,22 @@ where
         let signature = PUBKEY::sign(private_key, &message);
         Ok(Some(StaticVoteToken { signature, pub_key }))
     }
+    fn has_stake(&self, pub_key: &PUBKEY) -> bool {
+        let entry = pub_key.get_stake_table_entry(1u64);
```

Comment:
Is this line serves as assigning stake value (1u64) to a public key to make them searchable in committee? I feel rather than doing this, we'd better propagate in a complete node info (pub key + stake value) through the function parameter or design the committee only saving the pub key (which looks more reasonable), so that stake value is not something we assign. How do you think? We can leave it as a future issue, not an urgent thing for now.

Reply:
We could, but I don't want to hard code 1u64 throughout the code and we have no way to look up stake from the stake table.  We could change the static committee to to only save the public key.  I do want to rework the `get_stake_table_entry` function a bit but I think we should leave this as future work.

Reply:
Yeah I was trying to remove `get_stake_table_entry` and all these kinds of `1u64` in validator onboarding task, therefore added [this](https://github.com/EspressoSystems/HotShot/pull/1990/files#diff-cbe058cb32b3323389374453c6b2db7d52432aacf1a7e59d3ee2ee99364bccc4R294) so that stake value 1 will be a default setting. However not sure at which middle point this value together with the pub key are not propagated together :( But yeah this could be a future issue.

Reply:
I agree that this should be a future issue to clean up and make sure we are fetching the stake properly from the table in the production code and not using `get_stake_table_entry` (except in tests)